### PR TITLE
LicenseCheckBearTest: Use with to close file

### LINKED ISSUE #1704: Use with to load the file
In https://github.com/coala/coala-bears/blob/master/tests/general/LicenseCheckBearTest.py#L18,
file is opened without using `with` statement. It doesn't ensure the closing of file.

### PR DESCRIPTION
**Pull request by ragvri**
Fixes #1704

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by ragvri** [comment]
@jayvdb I am a complete beginner. How do I fix this issue of shortlog?

**Comment by jayvdb** [comment]
Please stop closing PRs, deleting your fork, etc.
Follow http://coala.io/newcomer - first step is come onto our Gitter chat room and say hello, so we can help you with each step.

File: tests/general/LicenseCheckBearTest.py [hunk 1]
```diff
@@ -15,7 +15,9 @@ def get_testfile_path(name):
 
 
 def load_testfile(name):
-    return open(get_testfile_path(name)).readlines()
+    with open(get_testfile_path(name), 'r') as f:
```
Comment (meetmangukiya) [inline tests/general/LicenseCheckBearTest.py]: Default is read mode, not required

**Comment by jayvdb** [comment]
ack 9ee9eeb

**Comment by jayvdb** [comment]
@rultor merge

**Pull request merged**